### PR TITLE
Rename formica to inago

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/547

```
$ git grep -i formica | wc -l
       0
$ git ls-files | grep -i formica | wc -l
       0
```

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
RFR @giantswarm/cena

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# formica
+# inago
```
Comment (xh3b4sd) [inline README.md]: Do we want to use title case? Inago is a name, so may `Inago` would be more appropriate when using it in free text.

Comment (JosephSalisbury) [inline README.md]: Yeah, that sounds good.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
-formicactl
+inagoctl
```
Comment (zeisss) [inline .gitignore:1]: can we temporariyl leave the old one in? otherwise I guarantee someone will commit his old binary.

**Comment by zeisss** [comment]
LGTM, although I would leave formicactl in the ignore list.

File: .gitignore (hunk 2 above)
Comment (JosephSalisbury) [inline .gitignore:1]: I'd rather not, it means we'll have it hanging around. I'm sure we'll notice in the PR if someone is committing binaries.

File: common/common.go [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // Package common provides implementation of general interest for certain sub
-// packages of formica.
+// packages of inago.
```
Comment (xh3b4sd) [inline common/common.go]: Title case.

File: controller/status.go [hunk 4]
```diff
@@ -171,7 +171,7 @@ var (
 )
 
 // AggregateStatus aggregates the given fleet and systemd states to a Status
-// known to formica based on the StatusIndex.
+// known to inago based on the StatusIndex.
```
Comment (xh3b4sd) [inline controller/status.go]: Title case.

File: docs/README.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # docs
-Here we document important concepts and functionality formica supports.
+Here we document important concepts and functionality inago supports.
```
Comment (xh3b4sd) [inline docs/README.md]: Title case.

File: .gitignore (hunk 2 above)
Comment (xh3b4sd) [inline .gitignore:1]: +1 @JosephSalisbury

**Pull request merged**